packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -174,9 +178,10 @@ class CohereProvider(BaseProvider, Cohere):
 
 class HfHubProvider(BaseProvider, HuggingFaceHub):
     id = "huggingface_hub"
-    name = "HuggingFace Hub"
+    name = "Hugging Face Hub"
     models = ["*"]
     model_id_key = "repo_id"
+    model_help = "See https://huggingface.co/models for a list of models."
```

Comment:
1. Rename this key to just `help`. This string isn't really specific to any model but rather to a provider, hence the current name isn't fully accurate.

2. I'd like this to be more explicit. Something like this:

> The Hugging Face Hub provider expects a model's HF Hub [repository ID](https://huggingface.co/docs/huggingface_hub/guides/repository) to be passed via `model_id`, e.g. `google/flan-t5-xl`. The full list of available models can be found on [Hugging Face](https://huggingface.co/models).